sotodlib/coords/det_match.py

```diff
@@ -687,11 +699,28 @@ def _get_biadjacency_matrix(self):
         m = src_arr['is_north'][:, None] != dst_arr['is_north'][None, :]
         mat[m] = np.inf
 
-        if self.match_pars.force_pointing:
-            m = np.isnan(src_arr['xi']) | np.isnan(src_arr['eta'])
-            mat[m, :] = np.inf
-            m = np.isnan(dst_arr['xi']) | np.isnan(dst_arr['eta'])
-            mat[:, m] = np.inf
+        if self.match_pars.enforce_pointing_reqs:
```

Comment:
So in this block, 'DARK' and 'SLOT' are implicitly untouched.  I think you need to at least _comment on_ DARK/SLOT, if not check explicitly that everything you haven't directly modified is listed as DARK or SLOT.